prompt

```diff
@@ -2,6 +2,5 @@
 # Opens a basic yes/no prompt with dmenu
 # This is useful for confirming whether an action should be taken
 
-if [ "$(printf "No\nYes" | dmenu -i -p "$1" -fn "JetBrains Mono Medium-10")" = "Yes" ]; then
-   $2
-fi
+[ "$(printf "No\nYes" | dmenu -i -p "$1" -fn "JetBrains Mono Medium-10")" = "Yes" ] && "$2"
```

Comment:
```suggestion
[ "$(printf 'No\nYes' | dmenu -i -p "$1" -fn 'JetBrains Mono Medium-10')" = 'Yes' ] && "$2"
```

You know, since we're being specific, here. lol :P

Reply:
yes, you're right, excuse me xD